src/ModAttachr.ts

```diff
@@ -104,15 +112,16 @@ export class ModAttachr implements IModAttachr {
         if (this.itemsHolder) {
             this.itemsHolder.setItem(this.transformModName(name), true);
         }
-
-        if (mod.events.hasOwnProperty("onModEnable")) {
+
+        if (this.eventNames.onModEnable) {

```

Comment:
This just checks whether `this.eventNames.onModEnable` exists (which is always true). It should be checking if such an event exists in the mod.